packages/ra-ui-materialui/package.json

```diff
@@ -49,6 +49,7 @@
         "classnames": "~2.2.5",
         "connected-react-router": "^6.4.0",
         "css-mediaquery": "^0.1.2",
+        "downshift": "3.2.7",
```

Comment:
`react-autosuggest` isn't removed because we still have AutocompletearrayInput, right?

Reply:
Yes, we'll have to rewrite this component too